packages/apps/kadena-transfer/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I see it's bootstrapped with Create React App. We are only using nextjs for projects to reduce maintenance burden. With nextjs we can use so called "static exports" to generate a static site. See https://nextjs.org/docs/pages/building-your-application/deploying/static-exports for more info